package.json

```diff
@@ -60,6 +60,7 @@
     "test": "run-s test:**",
     "test:snapshots": "jest --testPathPattern=\"storyshots.test.js\" --silent",
     "test:snapshots:update": "jest --testPathPattern=\"storyshots.test.js\" -u --silent",
+    "test:interaction": "jest --config jest-unit.config.js",
```

Comment:
Is it worth naming this config something else as we have a script of `test:unit` which just runs unit tests for the lambdas currently, https://github.com/yldio/yld.io/pull/322/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R66

Reply:
Right, I thought the same but then forgot to actually change the file name 😛